src/daemon/daemon.h

```diff
@@ -138,6 +138,8 @@ public slots:
     void release_resources(const std::string& instance);
     std::string check_instance_operational(const std::string& instance_name) const;
     std::string check_instance_exists(const std::string& instance_name) const;
+    MemorySize get_image_size(const VMImage& image);
+    MemorySize compute_final_image_size(const VMImage& image, multipass::optional<MemorySize> memory_argument);
```

Comment:
Couldn't these be in anonymous namespace instead? They're utility functions that I don't believe have any dependency on the Daemon object?

Reply:
Right, they could be in namespace{}.